synapse/lib/aha.py

```diff
@@ -916,8 +1153,18 @@ async def genCaCert(self, network):
 
         return cacert
 
+    async def _genCaCert(self, network):
+
+        if os.path.isfile(os.path.join(self.dirn, 'certs', 'cas', f'{network}.crt')):
```

Comment:
This seems like we're duplicating the same check that is done by the `genCaCert` function here? I don't know why we would need this to be separate.

Reply:
This check avoids the IO overhead of actually reading in the file.

Reply:
`self.certdir.getCaCertPath(network)` already exists to check this then.